core/src/types/capability.rs

```diff
@@ -155,6 +153,9 @@ pub struct Capability {
 
     /// If operator supports blocking.
     pub blocking: bool,
+
+    /// If operator supports versioning
+    pub versioning: bool,
```

Comment:
This is a breaking change that may require an RFC for discussion.

OpenDAL used to split these features into different operations, but there are issues as we add more features that span various operations. We can open a new issue to discuss it and determine if it's better to change our existing pattern.

In this PR, I suggest to keep it unchanged.